integrations/repofiles_delete_test.go

```diff
@@ -111,6 +112,8 @@ func testDeleteRepoFileWithoutBranchNames(t *testing.T, u *url.URL) {
 	test.LoadRepoCommit(t, ctx)
 	test.LoadUser(t, ctx, 2)
 	test.LoadGitRepo(t, ctx)
+	defer ctx.Repo.GitRepo.Close()
```

Comment:
How about add `MockContext.Close` and close repository there?

Reply:
We would need to add a Close() to context.Context as MockContext actually produces a real context.